### DISCUSSION TITLE
Packages/com.unity.inputsystem/manual/Workflows

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#595] [Notes]
# Packages/com.unity.inputsystem/manual/Workflows

https://docs.unity3d.com/Packages/com.unity.inputsystem@latest/index.html?subfolder=%2Fmanual%2FWorkflows.html

**Comment by marcospgp** [comment]
Don't forget action maps need to be enabled
